cms/static/js/views/modals/base_modal.js

```diff
@@ -55,6 +62,38 @@ define(['jquery', 'underscore', 'gettext', 'js/views/baseview'],
                 this.parentElement = parentElement;
             },
 
+            bindTabHandlers: function() {
```

Comment:
@andy-armstrong @cptvitamin I have tried to fix the tabbing behaviour on modal dialogs but things are not working as expected. Chrome, Firefox and Safari all show different behaviours. Please see the tab behaviour on modal dialogs [here](https://studio-upload-error-message.sandbox.edx.org/container/block-v1:edX+DemoX+Demo_Course+type@vertical+block@vertical_0270f6de40fc). Currently there are three dialogs for EDIT, Visibility Settings and Move.

Any suggestion to improve tab switch behaviour on these modals?